src/components/NavigationTabsComponent.jsx

```diff
@@ -0,0 +1,236 @@
+    currentUser, signOut, isLogged,
+    contactUsButton, showRequestModal, supportrequestModal,
+    tabs, initialTab, initialSubTab,
+    onSubtabChange, showProfileLinks, profileState
+  } = props;
+  const [selectedMenuTab, setSelectedMenuTab] = useState(false);
+  const [selectedSubTab, setSelectedSubTab] = useState(false);
+
+  useEffect(() => {
+    setSelectedMenuTab(initialTab === -1 ? false : initialTab)
+    setSelectedSubTab(initialSubTab === -1 ? false : initialSubTab)
+  }, [initialTab, initialSubTab]);
+
+  return (
+    <div className={`navbar-logged ${orientation === 'vertical' ? 'navbar-vertical' : ''}`}>
+      {makeNotifications()}
+      <ul className="navbar-main">
+        <div style={{ width: '100%', display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
+          {
+            orientation === 'horizontal' && (
+              <Link
+                id="link_logo"
+                to="/home"
+                className="navbar-brand"
+              >
+                <img style={duosLogoImage} src={DuosLogo} alt="DUOS Logo"/>
+              </Link>
+            )
+          }
+          {
+            isLogged && (
+              <Box className="duos-navigation-box">
+                <Tabs
+                  value={selectedMenuTab}
+                  variant="scrollable"
+                  scrollButtons="auto"
+                  orientation={orientation}
+                  TabIndicatorProps={{ style: { background: '#2BBD9B' } }}
+                >
+                  {tabs.map((tab, tabIndex) => (
+                    <Tab
+                      key={`${tab.link}_${tabIndex}`}
+                      label={tab.label}
+                      style={selectedMenuTab === tabIndex ? styles.mainTabActive : styles.mainTab}
+                      to={{ pathname: tab.link, state: { selectedMenuTab: tabIndex } }}
+                      component={Link}
+                    />
+                  ))}
+                </Tabs>
+              </Box>
+            )
+          }
+          {
+            !isLogged && (
+              <ul className="navbar-public">
+                <li>
+                  <a
+                    id="link_about"
+                    className="navbar-duos-link"
+                    href="https://support.terra.bio/hc/en-us/articles/28485372215579-About-DUOS"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    <div className="navbar-duos-icon-about" style={navbarDuosIcon}></div>
+                    <span style={navbarDuosText}>About</span>
+                  </a>
+                </li>
+                <li>
+                  <a
+                    id="link_help"
+                    className="navbar-duos-link"
+                    href="https://support.terra.bio/hc/en-us/articles/28486067349531-Frequently-Asked-Questions-about-DUOS"
+                    target="_blank" rel="noreferrer"
```

Comment:
```suggestion
                  >
                    <div className="navbar-duos-icon-about" style={navbarDuosIcon}></div>
                    <span style={navbarDuosText}>About</span>
                  </a>
                </li>
                <li>
                  <a
                    id="link_help"
                    className="navbar-duos-link"
                    href="https://support.terra.bio/hc/en-us/articles/28486067349531-Frequently-Asked-Questions-about-DUOS"
                    target="_blank"
```

We can safely remove `rel="noreferrer"`, I strongly suspect.

It was a security best practice around 8 years ago.  For around 4 years, [browsers have had built-in mitigation](https://mathiasbynens.github.io/rel-noopener/) for the exploit that motivated using `rel="noreferrer"` wherever `target="_blank"` is used.

Reply:
So, that's actually a linting issue! Specifically because old browsers might not have that setting. I think it doesn't hurt us to have it, and we can pull the rule later if we want.